pageserver/src/http/routes.rs

```diff
@@ -204,6 +206,8 @@ struct TimelineInfo {
     timeline_state: Option<TimelineSyncState>,
 }
 
+// TODO should we return something meaningful about remote timelines too
+// No way currently to ckeck uploading by tenant state, it woud be just active or idle (check by s3 consistent lsn?)
```

Comment:
```suggestion
// No way currently to check uploading by tenant state, it would be just active or idle (check by s3 consistent lsn?)
```

There would be a way very soon, see the last file in the PR for the API usage:
https://github.com/zenithdb/zenith/pull/991